docs/src/pages/guides/split-pages/sections/how-to.md

```diff
@@ -0,0 +1,70 @@
+---
+title: How to use spit pages
+description: An introduction to use split pages.
+order: 2
+splitPage: true
+products:
+  - Mapbox fundamentals
+prependJs:
+  - "import Note from '../../../../../../src/components/note/note';"
+  - "import MainPage from '!raw-loader!../index.js';"
+  - "import SubOne from '!raw-loader!./intro.md';"
+  - "import SplitPageShell from '!raw-loader!../../../../components/split-page-shell.js';"
+  - "import CodeSnippet from '../../../../../../src/components/code-snippet/code-snippet';"
+  - "import { highlightJsx } from '../../../../../../src/components/highlight/jsx';"
+  - "import { highlightHtml } from '../../../../../../src/components/highlight/html';"
+---
+
+## How to use split pages
+
+### 1. Create the main page
+
+The main page is where all the partial markdown files will be combined and displayed.
+
+- Create an `.js` file. The main file must be a JavaScript file to make sure the scroll spy on the sidebar works correctly.
+- Add `splitPages: true` to the frontMatter.
+- Add `order` to designate the order of appears in the `NavgiationAccordion`.
+- Import each markdown file in the main page.
+- Import the splitPages function to override the page's headings.
+- Set `hideFeedback: true` in the frontmatter of the main page (usually).
+
+{{ <div className="mb18"><CodeSnippet code={`${MainPage}`} highlighter={() => highlightJsx} filename="src/pages/guides/split-pages/index.js" /></div>}}
+
+### 2. Create the split pages
+
+You must save the partial markdown files in a folder adjacent to the main JavaScript page.
+
+- Create a folder, `sections`, adjacent to the main page.
+- Create markdown file for each page.
+- Add `splitPage: true` to the frontMatter.
+- Add `order` to designate the order the page appears.
+
+{{ <CodeSnippet code={`${SubOne}`} highlighter={() => highlightHtml} filename="src/pages/guides/split-pages/sections/intro.md" />}}
+
+### 3. Create split-page-shell
+
+You will need a wrapper to handle each split page, this is usually called `src/components/split-page-shell.js`.
```

Comment:
I know there's probably a good reason this is not made into a shared component, but I can't figure out why. 🤔  Is there something about this component that requires that it live in the repo that contains the content rather than being imported from dr-ui?

Reply:
This was on my to do list and I forgot to follow up. I think we could definitely create a shared component for this. I'll add it!